src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1971,7 +1983,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 
 	private _setAriaLabel(xterm: XTermTerminal | undefined, terminalId: number, title: string | undefined): void {
 		if (xterm && xterm.textarea) {
-			let label: string;
+			let label: string = this._accessibilityService.isScreenReaderOptimized() ? nls.localize('terminalTextBoxAriaLabelNumberAndTitleAccessibility', "Press alt+f1 for terminal accessibility help") : '';
```

Comment:
This will get overwritten by setting `label` below, we want to combine these messages, with the previous message first. Eg. `Terminal {0}, {1}. Press alt+f1 for terminal accessibility help`